### ISSUE TITLE
Build fails due to missing RoslynTools.MSBuild 16.0.0-alpha

### ISSUE THREAD

**Issue by ArrowCase** [dotnet/winforms#628]
I wasn't sure whether this issue belongs here or in [dotnet/arcade](https://github.com/dotnet/arcade). Please feel free to move it.

Steps to reproduce: Clone repository, run build.cmd.

Results:

```
dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Sdk/3.0.100-preview3-010436/dotnet-sdk-3.0.100-preview3-010436-win-x64.zip
dotnet-install: Extracting zip from https://dotnetcli.azureedge.net/dotnet/Sdk/3.0.100-preview3-010436/dotnet-sdk-3.0.100-preview3-010436-win-x64.zip
dotnet-install: Adding to current process PATH: "C:\Users\user\source\repos\winforms\.dotnet\". Note: This change will not be visible if PowerShell was run as a child process.
dotnet-install: Installation finished
Downloading vswhere
Downloading RoslynTools.MSBuild 16.0.0-alpha
No package matching package identifier RoslynTools.MSBuild and version 16.0.0-alpha was found in the given feed.
```

Version 16.0.0-alpha of the package indeed [does not appear to exist](https://dotnet.myget.org/F/roslyn-tools/api/v2/package/RoslynTools.MSBuild/16.0.0-alpha), but [16.0.0-rc1-alpha does](https://dotnet.myget.org/feed/roslyn-tools/package/nuget/RoslynTools.MSBuild).

Additionally, running `build` a second time results in a different error:

```
Exception calling "Start" with "0" argument(s): "The system cannot find the file specified"
System.Management.Automation.MethodInvocationException: Exception calling "Start" with "0" argument(s): "The system cannot find the file specified" ---> System.ComponentModel.Win32Exception: The system cannot find the file specified
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at CallSite.Target(Closure , CallSite , Object )
   --- End of inner exception stack trace ---
   at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext funcContext, Exception exception)
   at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
at Exec-Process, C:\Users\user\source\repos\winforms\eng\common\tools.ps1: line 69
at MSBuild, C:\Users\user\source\repos\winforms\eng\common\tools.ps1: line 476
at InitializeToolset, C:\Users\user\source\repos\winforms\eng\common\tools.ps1: line 422
at Build, C:\Users\user\source\repos\winforms\eng\common\Build.ps1: line 76
at <ScriptBlock>, C:\Users\user\source\repos\winforms\eng\common\Build.ps1: line 125
at <ScriptBlock>, <No file>: line 1
```

**Comment by zsd4yr** [comment]
@OliaG @merriemcgaw would one of you kindly move this to dotnet/arcade ?

**Comment by zsd4yr** [comment]
thanks @ArrowCase I am actively working on a workaround for us here but this is an underlying Arcade issue

**Comment by zsd4yr** [comment]
In case this ever happens again, the issue is that we're looking for 16.0.0 but there is no entry for it anymore https://dotnet.myget.org/feed/roslyn-tools/package/nuget/RoslynTools.MSBuild --see Package history

**Comment by zsd4yr** [comment]
Needs to be switched to a valid version; example: `16.0.0-rc1-alpha`

**Comment by zsd4yr** [comment]
This should be solved by a workaround on our end as part of #647
